sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -195,7 +195,7 @@ object DataSourceV2Strategy extends Strategy with PredicateHelper {
     case desc @ DescribeNamespace(catalog, namespace, extended) =>
       DescribeNamespaceExec(desc.output, catalog, namespace, extended) :: Nil
 
-    case desc @ DescribeTable(DataSourceV2Relation(table, _, _), isExtended) =>
+    case desc @ DescribeTable(DataSourceV2Relation(table, _, _, _, _), isExtended) =>
```

Comment:
Cases like this could be translated to `DescribeTable(rel: DataSourceV2Relation, isExtended)`. Then we wouldn't need to change this matcher every time the relation changes. Might be a good idea to rewrite it that way now.